modules/libarchive/3.7.4.bcr.2/source.json

```diff
@@ -0,0 +1,10 @@
+{
+    "integrity": "sha256-+Id1XENKc2pgnL0o2H3b++nWo7tbcDwiwC9q+AqAJzU=",
+    "strip_prefix": "libarchive-3.7.4",
+    "url": "https://github.com/libarchive/libarchive/releases/download/v3.7.4/libarchive-3.7.4.tar.xz",
+    "patch_strip": 1,
+    "patches": {
+        "build_with_bazel.patch": "sha256-/AAROymfaEnTk8XP0rLtgXijRdFqDNzCS1gf1lEtWXA=",
```

Comment:
I believe you can give a relative repo path to the previous patch instead of checking it in again

Reply:
I can't because it includes patching the `MODULE.bazel` file which has a different version.

Reply:
Ah, gah, I should have split it up more :) good to know for the future, thanks for explaining